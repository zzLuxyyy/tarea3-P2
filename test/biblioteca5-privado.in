crearBiblioteca
crearSocio 111 Ana Garcia 01 01 2000
agregarSocioBiblioteca
crearSocio 222 Bruno Fernandez 02 02 2001
agregarSocioBiblioteca
crearSocio 333 Carla Rodriguez 03 03 2002
agregarSocioBiblioteca
crearSocio 444 Daniel Lopez 04 04 2003
agregarSocioBiblioteca
crearFecha 01/01/1949
crearLibro 1001 "El Aleph" "Jorge Luis" "Borges" "Coleccion de cuentos" 1
agregarLibroBiblioteca
crearFecha 01/01/1944
crearLibro 1002 "Ficciones" "Jorge Luis" "Borges" "Otra coleccion de cuentos" 1
agregarLibroBiblioteca
crearFecha 01/09/2025
prestarLibroBiblioteca 111 1001
crearFecha 02/09/2025
prestarLibroBiblioteca 222 1002
imprimirPrestamosBiblioteca
reservarLibroBiblioteca 222 1001
imprimirReservasBiblioteca
crearFecha 15/09/2025
devolverLibroBiblioteca 111 1001 01/09/2025
imprimirPrestamosBiblioteca
imprimirReservasBiblioteca
liberarBiblioteca
Fin