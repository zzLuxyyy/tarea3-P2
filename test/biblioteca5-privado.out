1>Biblioteca creada exitosamente.
2>Socio de ci 111 creado exitosamente.
3>Socio de CI 111 agregado a la biblioteca.
4>Socio de ci 222 creado exitosamente.
5>Socio de CI 222 agregado a la biblioteca.
6>Socio de ci 333 creado exitosamente.
7>Socio de CI 333 agregado a la biblioteca.
8>Socio de ci 444 creado exitosamente.
9>Socio de CI 444 agregado a la biblioteca.
10>11>12>Libro de ISBN 1001 agregado a la biblioteca.
13>14>15>Libro de ISBN 1002 agregado a la biblioteca.
16>17>Préstamo del libro 1001 al socio 111 registrado.
18>19>Préstamo del libro 1002 al socio 222 registrado.
20>LDE Préstamos:
Préstamo de libro El Aleph a Ana Garcia.
1/9/2025
No retornado
Préstamo de libro Ficciones a Bruno Fernandez.
2/9/2025
No retornado
21>Reserva del libro 1001 para el socio 222 registrada.
22>Cola de Reservas:
Reserva de libro El Aleph por Bruno Fernandez.
23>24>Devolución del libro 1001 por el socio 111 registrada.
25>LDE Préstamos:
Préstamo de libro El Aleph a Ana Garcia.
1/9/2025
15/9/2025
Préstamo de libro Ficciones a Bruno Fernandez.
2/9/2025
No retornado
26>Cola de Reservas:
Reserva de libro El Aleph por Bruno Fernandez.
27>Biblioteca liberada exitosamente.
28>Fin.
